### PR TITLE
Add company_code to response parameters

### PR DESCRIPTION
**Pull request by ku00**
I added one response parameter to response as `company_code` .

This code is necessary when pay by family_mart.

### CODE REVIEW

File: test/remote/gateways/remote_epsilon_convenience_store_test.rb [hunk 1]
```diff
@@ -16,6 +16,7 @@ def test_convenience_store_purchase_successful
       assert_match /\d{4}\-\d{2}\-\d{2} \d{2}:\d{2}:\d{2}/, response.params['receipt_date']
       assert_match /\d{4}\-\d{2}\-\d{2}/, response.params['convenience_store_limit_date']
       assert_match %r!\Ahttp://.+!, response.params['convenience_store_payment_slip_url']
+      assert_match /\d{5}/, response.params['receipt_number']
```
Comment (kurotaky) [inline test/remote/gateways/remote_epsilon_convenience_store_test.rb]: receipt_number params already compared at Line 15.
https://github.com/pepabo/active_merchant-epsilon/blob/add-response-params/test/remote/gateways/remote_epsilon_convenience_store_test.rb#L15
 Is this correct?

Comment (ku00) [inline test/remote/gateways/remote_epsilon_convenience_store_test.rb]: Oh, I'm sorry...!
This is wrong, I really wanted to write `response.params['company_code']` .

Comment (kurotaky) [inline test/remote/gateways/remote_epsilon_convenience_store_test.rb]: OK!

**Comment by kurotaky** [comment]
@ku00 
Released. Thanks!
:rocket: https://rubygems.org/gems/active_merchant-epsilon/versions/0.5.6

**Comment by ku00** [comment]
Oh..., I noticed the mistake at Line 73. 
This response doesn't parsed correctly.

https://github.com/pepabo/active_merchant-epsilon/blob/cb7c8aa3a330317e44f83f66f93f107510a319c2/lib/active_merchant/billing/gateways/response_parser.rb#L73

So,  should remove one code `@xml.xpath` . This code is written just one.

**Pull request merged**